hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommitLocks.java

```diff
@@ -72,10 +79,15 @@ public class TestHiveCommitLocks extends HiveTableBaseTest {
   LockResponse notAcquiredLockResponse = new LockResponse(dummyLockId, LockState.NOT_ACQUIRED);
 
   @BeforeClass
-  public static void initializeSpies() throws Exception {
+  public static void startMetastore() throws Exception {
+    HiveMetastoreTest.startMetastore(Collections.singletonMap(HiveConf.ConfVars.HIVE_TXN_TIMEOUT.varname, "1s"));
```

Comment:
I think in the Iceberg code we usually use: `org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap.of`